cutouts_channels.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python -u
+
+#############################################################################################
+#
+# Python script to demonstrate interacting with CASDA's TAP and SODA implementations to
+# retrieve cutout images in bulk and slice them up by channels.
+#
+# This script does a TAP query to get the image cubes for a given scheduling block, and can be
+# configured to either:
+# a) conduct a second TAP query to identify catalogue entries of interest, and create an async
+#    job to download cutouts at the RA and DEC of each of the catalogue entries.
+# b) create an async job to download the entire image cube file.
+#
+# Author: Chris Trapani on 3 July 2019
+#
+# Written for python 3.7
+# Note: astropy is available on galaxy via 'module load astropy'
+# On other machines, try Anaconda https://www.continuum.io/downloads
+#
+#############################################################################################
+
+from __future__ import print_function, division, unicode_literals
+
+import argparse
+import os
+import astropy.units as u
+import math
+
+from astropy.io.votable import parse
+
+
+import casda
+
+def parseargs():
+    """
+    Parse the command line arguments
+    :return: An args map with the parsed arguments
+    """
+    parser = argparse.ArgumentParser(
+        description="Generate and download cutouts around each source identified in a scheduling block.")
+    parser.add_argument("opal_username",
+                        help="Your user name on the ATNF's online proposal system (normally an email address)")
+    parser.add_argument("-p", "--opal_password", help="Your password on the ATNF's online proposal system")
+    parser.add_argument("--password_file", help="The file holding your password for the ATNF's online proposal system")
+    parser.add_argument("scheduling_block_id", help="The id of the ASKAP scheduling block to be queried.")
+    parser.add_argument("num_channels", help="Number of channels to slice each cube by.")
+    parser.add_argument("-type", "--data_product_type", help="Sub-type of the data product. E.g. 'spectral.restored.3d'.  "
```

Comment:
You can set a default here, which will save the if test later in the code default='spectral.restored.3d'